extensions/machine-learning/src/test/views/utils.ts

```diff
@@ -16,6 +16,7 @@ export interface ViewTestContext {
 
 export function createViewContext(): ViewTestContext {
 	let onClick: vscode.EventEmitter<any> = new vscode.EventEmitter<any>();
+	let onChange: vscode.EventEmitter<any> = new vscode.EventEmitter<any>();
```

Comment:
boolean for the types, not any

Reply:
(make sure to update all the other instances of this too)

Reply:
In future, if the same onChange is used for other components it will have a different type. Therefore, I left it as any.

Reply:
Not sure what you mean - each component should implement its own version of onChanged and thus can set the value to whatever is appropriate for that component. And anyways it's better to do the right thing now and then if someone does do something like that in the future it can be addressed now (likely by just making another emitter of the appropriate type)

Reply:
@aasimkhan30 ping - it's a bit of a nit but we really should be getting in the habit of avoiding using any